src/contrib/cluster/Akka.Cluster.Tools.Tests/Singleton/ClusterSingletonConfigSpec.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Akka.Cluster.Tools.Tests.Singleton
 {
+    // <ClusterSingletonConfigSpec>
```

Comment:
I am showing how the Test Class is derived from the `TeskKit`, and the custom config, `GetConfig()`, itself so that there are confusion about the setup!

Reply:
LGTM